apps/auth_content/models.py

```diff
@@ -109,3 +109,34 @@ def get_context(self, request):
         context["closedpod_children_pages"] = self.get_siblings()
         context["closedpod_home_url"] = self.get_parent().url
         return context
+
+
+class PCWMSAHomePage(StaticPage):
+    max_count = 1
```

Comment:
Wonder if we should limit parent types for this (and the closedpod page) to the Home Page?

Reply:
Sure, sounds like a good idea